packages/angular_devkit/build_angular/src/webpack/configs/styles.ts

```diff
@@ -162,7 +162,7 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
   // A configuration file can exist in the project or workspace root
   const tailwindConfigFile = 'tailwind.config.js';
   let tailwindConfigPath;
-  for (const basePath of [wco.projectRoot, wco.root]) {
+  for (const basePath of [wco.root, wco.projectRoot]) {
     const fullPath = path.join(basePath, tailwindConfigFile);
     if (fs.existsSync(fullPath)) {
```

Comment:
The problem here is that there is no break statement.

Reply:
Done!